packages/calcite-components/src/components/accordion/readme.md

```diff
@@ -1,6 +1,6 @@
 # calcite-accordion
 
-`calcite-accordion` can be used to present content in collapsible sections. Configuration options exist for visual style (icon-position, scale, appearance), and selection functionality.
+`calcite-accordion` can be used to present content in collapsible sections. Configuration options exist for visual style icon-type, icon-position, scale, appearance, and selection functionality.
```

Comment:
I don't think the second sentence is necessary. It's just listing prop names. I'd keep the first sentence.